.gitignore

```diff
@@ -16,3 +16,5 @@ package-lock.json
 .kitchen-sink/
 __pycache__
 doc/
+dist/
```

Comment:
Should we be adding dist/pbjs-genfiles to the gitignore?
We won't be directly webpacking gax I believe and pbjs-genfiles should be handled by the generator changes in the separate clients.

Reply:
Yes, everything that is not supposed to be checked in should go into `.gitignore`. Dist must be there since someone might eventually do `npx webpack` on gax.